js/curation-tool.js

```diff
@@ -897,6 +897,29 @@ function load_json_from_local(file_input) {
     fr.readAsText(file);
 }
 
+/**
+ * get_newick_string_from_phylotree(phylotree)
+ *
+ * Given a Phylotree object, convert it into a Newick string.
+ */
+function get_newick_string_from_phylotree(phylotree) {
```

Comment:
JavaScript makes it actually very easy to replace methods with customizations for applications:
```js
let getNewick = tree.get_newick;
tree.get_newick = function(annotFunc) {
    return getNewick.call(this, annotFunc) + ";";
};
```
And afterwards you treat phylotree.get_newick as having the behavior you desire (because it actually does). 

I'm using `tree` here and not`phylotree` because you would do this after the object is created (which I think is in `render_tree()`.

JavaScript uses objects and inheritance by prototyping. You can replace any method at any time with anything you wish.